test/lib/utils/StylingUtil.test.ts

```diff
@@ -0,0 +1,103 @@
+import * as fs from 'node:fs/promises';
+import path from 'node:path';
+import {toStyledHeaderAndBody, toStyledHeader, toStyledPropertyList} from '../../../src/lib/utils/StylingUtil';
+
+const PATH_TO_COMPARISON_FILES = path.resolve('.', 'test', 'fixtures', 'comparison-files', 'lib',
```

Comment:
May I suggest that we use `__dirname` instead of assuming the cwd whenever possible so that if we ever restructure tests (or run the tests from somewhere else other than the project root) then we won't be bitten. It isn't as big of a deal here, but in our monorepo this really is important.

Reply:
reasonable.